### ISSUE TITLE
[a11y] newsletter banner on releasenotes page needs a unique landmark label

### ISSUE THREAD

**Issue by alexgibson** [mozilla/bedrock#15958]
# Axe-core® Accessibility Results for /en-US/firefox/releasenotes/ (desktop) project

Page URL: [https://dev.bedrock.nonprod.webservices.mozgcp.net/en-US/firefox/135.0/releasenotes/](https://dev.bedrock.nonprod.webservices.mozgcp.net/en-US/firefox/135.0/releasenotes/)  

## axe-core found 1 violation

| #   | Description | Axe rule ID | WCAG | Impact | Count |
| --- | --- | --- | --- | --- | --- |
| [1](#1) | Landmarks should have a unique role or role/label/title (i.e. accessible name) combination | landmark-unique | Best practice | moderate | 1   |

### Failed

1. Landmarks should have a unique role or role/label/title (i.e. accessible name) combination

[Learn more](https://dequeuniversity.com/rules/axe/4.10/landmark-unique?application=playwright)

landmark-unique

Best practice

Ensure landmarks are unique

moderate

Issue Tags: cat.semantics best-practice

| #   | Issue Description | To solve this violation, you need to... |
| --- | --- | --- |
| 1   | **Element location**<br><br>```<br>.m24-pencil-banner<br>```<br><br>**Element source**<br><br>```<br><aside class="m24-pencil-banner" aria-label="Promotion" data-nosnippet="true"><br>``` | Fix any of the following:<br><br>*   The landmark must have a unique aria-label, aria-labelledby, or title to make landmarks distinguishable<br><br>**Related node:**<br><br>```<br>#newsletter-banner<br>``` |
